src/strings.rs

```diff
@@ -19,6 +19,15 @@ pub struct InlineStr {
     inner: [u8; MAX_INLINE_STR_LEN],
 }
 
+impl InlineStr {
+    // Returns an empty `InlineStr`.
+    pub fn new() -> Self {
```

Comment:
What do you think about renaming it to `pub fn empty()`? I'd expect `SomeStr::new` to take an argument which is converted to that `SomeStr`. /bikeshedding

Reply:
I think I prefer that, but picked `new` to match `String`. I can rename it, of course.